## **The Nobel Prize is perhaps the world's most recognized scientific award. It is awarded every year to scientists and academics in the fields of chemistry, literature, physics, medicine, economy and peace. In this project, we will examine Nobel Prize winners. **

### **Purpose of this anaylsis** : *It is an analysis to examine the awarding habits of Nobel Prizes and the cultural patterns of the awards.*

# **Query steps for analysis**

*   Which countries were the first to win the most Nobel Prizes?

*   List the first women to win Nobel Prizes.

*   List the first men to win Nobel Prizes.

*   From which year or decade did the country that won the most Nobel Prizes seen a rapid increase in the total number of awards?

*   What this dominance looks like and what are the factors that play a role in this dominance?

*   The Nobel Prize winners using their gender, age, award category and year.

*   The categories and countries of the Nobel Prize winners between 1938 and 1945.

*   The categories and countries of the Nobel Prize winners between 1947 and 1991.

*   The countries and ages of people after 2000 in the categories of Chemistry, Literature, Peace, Physics and Medicine.

## **STEPS TAKEN DURING CLEANING AND PREPARATION**

*   A connection to Google Drive has been established.
*   The libraries to be used have been loaded.
*   The data file has been loaded.
*   Information about the data was obtained about type, row information, column information, basic information (NaN values) and column types.
*   The totals in the columns of the empty values ​​were checked.
*   All columns that will not be used during analysis have been deleted.
*   The data types of the remaining columns were checked.
*   Again, how many rows in total and in which columns the NaN values ​​were checked.
*   While all the rows with empty values ​​were analyzed, they were deleted because they were not in high numbers and it was assumed that they would not cause any problems.
*   Ready for data analysis.






## **NOTE: Most of the queries have been visualized.**

## CLEANING AND PREPARATION



> exploratory data analysis



In [ ]:
from google.colab import drive
drive.mount('/content/drive')

Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount("/content/drive", force_remount=True).


In [ ]:
# Libraries to be used have been loaded

import pandas as pd
import matplotlib.pyplot as plt
import plotly.express as px

In [ ]:
# Reading Nobel csv excel file.

nobel = pd.read_csv('/content/asset-v1_Kodlasam+MS01+2023_06+type@asset+block@nobel.csv')
nobel.head(10)

year    category                                           prize  \
0  1901   Chemistry               The Nobel Prize in Chemistry 1901   
1  1901  Literature              The Nobel Prize in Literature 1901   
2  1901    Medicine  The Nobel Prize in Physiology or Medicine 1901   
3  1901       Peace                      The Nobel Peace Prize 1901   
4  1901       Peace                      The Nobel Peace Prize 1901   
5  1901     Physics                 The Nobel Prize in Physics 1901   
6  1902   Chemistry               The Nobel Prize in Chemistry 1902   
7  1902  Literature              The Nobel Prize in Literature 1902   
8  1902    Medicine  The Nobel Prize in Physiology or Medicine 1902   
9  1902       Peace                      The Nobel Peace Prize 1902   

                                          motivation prize_share  laureate_id  \
0  "in recognition of the extraordinary services ...         1/1          160   
1  "in special recognition of his poetic composit...         1/1          569   
2  "for his work on serum therapy, especially its...         1/1          293   
3                                                NaN         1/2          462   
4                                                NaN         1/2          463   
5  "in recognition of the extraordinary services ...         1/1            1   
6  "in recognition of the extraordinary services ...         1/1          161   
7  "the greatest living master of the art of hist...         1/1          571   
8  "for his work on malaria, by which he has show...         1/1          294   
9                                                NaN         1/2          464   

  laureate_type                           full_name  birth_date  \
0    Individual        Jacobus Henricus van 't Hoff  1852-08-30   
1    Individual                     Sully Prudhomme  1839-03-16   
2    Individual              Emil Adolf von Behring  1854-03-15   
3    Individual                   Jean Henry Dunant  1828-05-08   
4    Individual                      Frédéric Passy  1822-05-20   
5    Individual              Wilhelm Conrad Röntgen  1845-03-27   
6    Individual                Hermann Emil Fischer  1852-10-09   
7    Individual  Christian Matthias Theodor Mommsen  1817-11-30   
8    Individual                         Ronald Ross  1857-05-13   
9    Individual                       Élie Ducommun  1833-02-19   

           birth_city        birth_country   sex   organization_name  \
0           Rotterdam          Netherlands  Male   Berlin University   
1               Paris               France  Male                 NaN   
2   Hansdorf (Lawice)     Prussia (Poland)  Male  Marburg University   
3              Geneva          Switzerland  Male                 NaN   
4               Paris               France  Male                 NaN   
5  Lennep (Remscheid)    Prussia (Germany)  Male   Munich University   
6          Euskirchen    Prussia (Germany)  Male   Berlin University   
7             Garding  Schleswig (Germany)  Male                 NaN   
8              Almora                India  Male  University College   
9              Geneva          Switzerland  Male                 NaN   

  organization_city organization_country  death_date      death_city  \
0            Berlin              Germany  1911-03-01          Berlin   
1               NaN                  NaN  1907-09-07        Châtenay   
2           Marburg              Germany  1917-03-31         Marburg   
3               NaN                  NaN  1910-10-30          Heiden   
4               NaN                  NaN  1912-06-12           Paris   
5            Munich              Germany  1923-02-10          Munich   
6            Berlin              Germany  1919-07-15          Berlin   
7               NaN                  NaN  1903-11-01  Charlottenburg   
8         Liverpool       United Kingdom  1932-09-16    Putney Heath   
9               NaN                  NaN  1906-12-07            Bern   

    d

In [ ]:
# The process performed to obtain information about the table. Null values, types, column names, number of rows of data

nobel.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 911 entries, 0 to 910
Data columns (total 18 columns):
 #   Column                Non-Null Count  Dtype 
---  ------                --------------  ----- 
 0   year                  911 non-null    int64 
 1   category              911 non-null    object
 2   prize                 911 non-null    object
 3   motivation            823 non-null    object
 4   prize_share           911 non-null    object
 5   laureate_id           911 non-null    int64 
 6   laureate_type         911 non-null    object
 7   full_name             911 non-null    object
 8   birth_date            883 non-null    object
 9   birth_city            883 non-null    object
 10  birth_country         885 non-null    object
 11  sex                   885 non-null    object
 12  organization_name     665 non-null    object
 13  organization_city     667 non-null    object
 14  organization_country  667 non-null    object
 15  death_date            593 non-null    ob

In [ ]:
# column-wise empty data count list

nobel.isnull().sum()

year                      0
category                  0
prize                     0
motivation               88
prize_share               0
laureate_id               0
laureate_type             0
full_name                 0
birth_date               28
birth_city               28
birth_country            26
sex                      26
organization_name       246
organization_city       244
organization_country    244
death_date              318
death_city              335
death_country           329
dtype: int64

In [ ]:
# Finding lines that will not be useful to us while doing the analysis

nobel = nobel.drop (["motivation","death_country","death_city","death_date","organization_country","organization_city","birth_city","laureate_id","prize_share","organization_name","laureate_type"], axis = 1)
nobel.head(10)

year    category                                           prize  \
0  1901   Chemistry               The Nobel Prize in Chemistry 1901   
1  1901  Literature              The Nobel Prize in Literature 1901   
2  1901    Medicine  The Nobel Prize in Physiology or Medicine 1901   
3  1901       Peace                      The Nobel Peace Prize 1901   
4  1901       Peace                      The Nobel Peace Prize 1901   
5  1901     Physics                 The Nobel Prize in Physics 1901   
6  1902   Chemistry               The Nobel Prize in Chemistry 1902   
7  1902  Literature              The Nobel Prize in Literature 1902   
8  1902    Medicine  The Nobel Prize in Physiology or Medicine 1902   
9  1902       Peace                      The Nobel Peace Prize 1902   

                            full_name  birth_date        birth_country   sex  
0        Jacobus Henricus van 't Hoff  1852-08-30          Netherlands  Male  
1                     Sully Prudhomme  1839-03-16               France  Male  
2              Emil Adolf von Behring  1854-03-15     Prussia (Poland)  Male  
3                   Jean Henry Dunant  1828-05-08          Switzerland  Male  
4                      Frédéric Passy  1822-05-20               France  Male  
5              Wilhelm Conrad Röntgen  1845-03-27    Prussia (Germany)  Male  
6                Hermann Emil Fischer  1852-10-09    Prussia (Germany)  Male  
7  Christian Matthias Theodor Mommsen  1817-11-30  Schleswig (Germany)  Male  
8                         Ronald Ross  1857-05-13                India  Male  
9                       Élie Ducommun  1833-02-19          Switzerland  Male

In [ ]:
# types of all remaining columns

nobel.dtypes

year              int64
category         object
prize            object
full_name        object
birth_date       object
birth_country    object
sex              object
dtype: object

In [ ]:
# and this is a column wise total list of empty data in the remaining columns

nobel.isnull().sum()

year              0
category          0
prize             0
full_name         0
birth_date       28
birth_country    26
sex              26
dtype: int64

In [ ]:
# deleting all empty remaining columns

nobel.dropna(inplace=True)

In [ ]:
nobel.isnull().sum()

year             0
category         0
prize            0
full_name        0
birth_date       0
birth_country    0
sex              0
dtype: int64

## PROJE QUERIES

## Which countries were the first to win the most Nobel Prizes?

In [ ]:
#We group by on the basis of countries of birth and get the total number
# then sorting from largest to smallest, change the column name and rewrite the sorting numbers from the table.

top10 = nobel.groupby('birth_country').size().reset_index(name='Toplam').nlargest(10, 'Toplam').reset_index(drop=True)
top10

birth_country  Toplam
0  United States of America     258
1            United Kingdom      85
2                   Germany      61
3                    France      51
4                    Sweden      29
5                     Japan      24
6                    Canada      18
7               Netherlands      18
8                     Italy      17
9                    Russia      17

### Insight: According to this data, over the 115 years of data considered, the USA, the United Kingdom, Germany and France were the countries that received the most Nobel Prizes in total.
### The USA has a total of 258 Nobel Prizes in 6 different fields. The USA is equal to the total of the other 6 countries, respectively.
### It is clearly seen that the development of the USA in many areas, with the influence of the events of the last 100 years, has a direct impact on this score.



# List the first women to win Nobel Prizes.

In [ ]:
# It filters the entire table on men and prints the first 10 rows from smallest to largest. New index numbers are added to the resulting table.

nobel_male_firsttop10 = nobel[nobel['sex'] == 'Male'].nsmallest(10, 'year').reset_index(drop=True)
nobel_male_firsttop10

year    category                                           prize  \
0  1901   Chemistry               The Nobel Prize in Chemistry 1901   
1  1901  Literature              The Nobel Prize in Literature 1901   
2  1901    Medicine  The Nobel Prize in Physiology or Medicine 1901   
3  1901       Peace                      The Nobel Peace Prize 1901   
4  1901       Peace                      The Nobel Peace Prize 1901   
5  1901     Physics                 The Nobel Prize in Physics 1901   
6  1902   Chemistry               The Nobel Prize in Chemistry 1902   
7  1902  Literature              The Nobel Prize in Literature 1902   
8  1902    Medicine  The Nobel Prize in Physiology or Medicine 1902   
9  1902       Peace                      The Nobel Peace Prize 1902   

                            full_name  birth_date        birth_country   sex  
0        Jacobus Henricus van 't Hoff  1852-08-30          Netherlands  Male  
1                     Sully Prudhomme  1839-03-16               France  Male  
2              Emil Adolf von Behring  1854-03-15     Prussia (Poland)  Male  
3                   Jean Henry Dunant  1828-05-08          Switzerland  Male  
4                      Frédéric Passy  1822-05-20               France  Male  
5              Wilhelm Conrad Röntgen  1845-03-27    Prussia (Germany)  Male  
6                Hermann Emil Fischer  1852-10-09    Prussia (Germany)  Male  
7  Christian Matthias Theodor Mommsen  1817-11-30  Schleswig (Germany)  Male  
8                         Ronald Ross  1857-05-13                India  Male  
9                       Élie Ducommun  1833-02-19          Switzerland  Male

### Insight: The first thing to notice in the first men's winners list is the absence of the Literature category. In addition, the Peace Prize was given to 2 people in its first year.
### Besides, 1. In 2017, the award was given only to European countries.


# List the first men to win Nobel Prizes.

In [1]:
# The 3rd query and the 2nd query have the same code logic.

nobel_female_firsttop10 = nobel[nobel['sex'] == 'Female'].nsmallest(10, 'year').reset_index(drop=True)
nobel_female_firsttop10

NameError: name 'nobel' is not defined

### Insight: When we look at the list of the top 10 female award winners, it appears that the first award was awarded between 1903 and the 10th award in 1945, and that in 42 years there were significantly fewer female winners.
### One of the reasons why women received so little in the first 40-45 years may be the First and Second World Wars.
### It may also be that higher and primary education (reading and writing) is not provided equally between men and women.


# From which year or decade did the country that won the most Nobel Prizes seen a rapid increase in the total number of awards?

# What this dominance looks like and what are the factors that play a role in this dominance?

In [ ]:
# Same as the logic of the 2nd and 3rd queries, filtering was done on USA. The table is sorted by year from smallest to largest.
nobel_winner_usa = nobel[nobel['birth_country'] == 'United States of America'].reset_index(drop=True)
nobel_winner_usa

year    category                                           prize  \
0    1906       Peace                      The Nobel Peace Prize 1906   
1    1912       Peace                      The Nobel Peace Prize 1912   
2    1914   Chemistry               The Nobel Prize in Chemistry 1914   
3    1919       Peace                      The Nobel Peace Prize 1919   
4    1923     Physics                 The Nobel Prize in Physics 1923   
..    ...         ...                                             ...   
253  2014   Chemistry               The Nobel Prize in Chemistry 2014   
254  2014   Chemistry               The Nobel Prize in Chemistry 2014   
255  2014    Medicine  The Nobel Prize in Physiology or Medicine 2014   
256  2015   Chemistry               The Nobel Prize in Chemistry 2015   
257  2016  Literature              The Nobel Prize in Literature 2016   

                     full_name  birth_date             birth_country   sex  
0           Theodore Roosevelt  1858-10-27  United States of America  Male  
1                   Elihu Root  1845-02-15  United States of America  Male  
2    Theodore William Richards  1868-01-31  United States of America  Male  
3        Thomas Woodrow Wilson  1856-12-28  United States of America  Male  
4      Robert Andrews Millikan  1868-03-22  United States of America  Male  
..                         ...         ...                       ...   ...  
253                Eric Betzig  1960-01-13  United States of America  Male  
254         William E. Moerner  1953-06-24  United States of America  Male  
255               John O'Keefe  1939-11-18  United States of America  Male  
256               Paul Modrich  1946-06-13  United States of America  Male  
257                  Bob Dylan  1941-05-24  United States of America  Male  

[258 rows x 7 columns]

In [ ]:
#It will bring the categories that the list has won on a category basis.

nobel_winner_usa_grouped = nobel_winner_usa.groupby(['category']).size().reset_index(name='count')

nobel_winner_usa_grouped

category  count
0   Chemistry     52
1   Economics     43
2  Literature      9
3    Medicine     70
4       Peace     19
5     Physics     65

In [2]:
#Groups were made over the years with the histogram chart and the total number of Nobel Prize winners in each 10 years was calculated.

fig1 = px.histogram(nobel_winner_usa, x='year',title="USA'S DECADELICAL NOBEL WINNERS (1901-2016)")


fig1.update_layout(
    title=dict(text="USA'S DECADELICAL NOBEL WINNERS (1901-2016)", font=dict(size=24)),
)

fig1.show()

NameError: name 'px' is not defined

### Insight - When we look at the Nobel Prizes won by the USA, we see that they have increased rapidly and exponentially since 1930.

### One of the most important reasons is that after the economic and social problems it experienced in the 1920s, it received immigration due to the wars in Europe in the 1930s, there were developments in industry and it was at the center of development while other countries were at war. It can also be said that he should stay away from other negative activities.

### It can be said that the great brain drain caused by the world war in Europe in the 1940s was a major factor in people migrating to safer continents. The explanation for the small decline in the 1940s may be the World War, which had a very high impact on the US as well as the world population.

### It continues to reap the fruits of the great brain drain it received in its country in the 1950s and 1940s. During this period, there were also major developments in space science. The negative effects of the Veitnam War, which continued between 1955 and 1975, can be seen as a decline in 1980.

### This exponentially growing decline in the number of Nobel prizes continued until 2010.

### The fact that Europe has become the new center again in the last 10-15 years and that more immigration and education levels have increased in these lands have been a major factor.

# The Nobel Prize winners using their gender, age, award category and year.

In [ ]:
#first of all, a copy of the Nobel table was created. We need to subtract the year column from the birthdate column because we will make the column of earning age and we will derive an insight based on these numbers.
#birthdate data type was converted from string to birthdate type and subtraction was performed.

nobel1 = nobel.copy()
nobel1['birth_date'] = pd.to_datetime(nobel1['birth_date'], errors='coerce')
nobel1['winner_age'] = nobel1['year'] - nobel1['birth_date'].dt.year

nobel1

year    category                                           prize  \
0    1901   Chemistry               The Nobel Prize in Chemistry 1901   
1    1901  Literature              The Nobel Prize in Literature 1901   
2    1901    Medicine  The Nobel Prize in Physiology or Medicine 1901   
3    1901       Peace                      The Nobel Peace Prize 1901   
4    1901       Peace                      The Nobel Peace Prize 1901   
..    ...         ...                                             ...   
906  2016    Medicine  The Nobel Prize in Physiology or Medicine 2016   
907  2016       Peace                      The Nobel Peace Prize 2016   
908  2016     Physics                 The Nobel Prize in Physics 2016   
909  2016     Physics                 The Nobel Prize in Physics 2016   
910  2016     Physics                 The Nobel Prize in Physics 2016   

                        full_name birth_date     birth_country   sex  \
0    Jacobus Henricus van 't Hoff 1852-08-30       Netherlands  Male   
1                 Sully Prudhomme 1839-03-16            France  Male   
2          Emil Adolf von Behring 1854-03-15  Prussia (Poland)  Male   
3               Jean Henry Dunant 1828-05-08       Switzerland  Male   
4                  Frédéric Passy 1822-05-20            France  Male   
..                            ...        ...               ...   ...   
906              Yoshinori Ohsumi 1945-02-09             Japan  Male   
907            Juan Manuel Santos 1951-08-10          Colombia  Male   
908             David J. Thouless 1934-09-21    United Kingdom  Male   
909          F. Duncan M. Haldane 1951-09-14    United Kingdom  Male   
910         J. Michael Kosterlitz 1943-06-22    United Kingdom  Male   

     winner_age  
0            49  
1            62  
2            47  
3            73  
4            79  
..          ...  
906          71  
907          65  
908          82  
909          65  
910          73  

[883 rows x 8 columns]

In [3]:

#A scatter plot chart was made by putting the year of acquisition on the x-axis and the age at acquisition on the y-axis of the data in the table above.
#Male and Female winners were shown on two separate lines.
#categories are shown in columns.
#spelling order was also specified.
#and title added.

fig2 = px.scatter(nobel1, x="year", y='winner_age', color='sex', height=700, width=3000,
                  facet_row='sex', facet_col="category",
                  category_orders={"sex": ["Male", "Female"],
                                   "category": ["Chemistry", "Literature", "Medicine", "Peace", "Physics", "Economics"]},
                  labels={"sex": "Cinsiyet", "category": "Kategori", "year": "Yıl", "winner_age": "Kazanan Yaş"},
                  title="Distribution of Nobel Prize Winners by Age and Category")

fig2.update_layout(
    title=dict(text="Distribution of Nobel Prize Winners by Age and Category", font=dict(size=24)),
)

fig2.show()

NameError: name 'px' is not defined

### Insight - If all categories are evaluated one by one;

### It can be seen from the distribution chart that in the Chemistry category, the number of male winners is 10 or 11 times more than female winners. It seems that the number of female winners in 115 years is 4. There does not seem to be an award for women scientists in this regard in the last 100 years, and the winners in this category appear to be between the ages of 40 and 80.

### It is seen that women received many awards in the field of literature in the 90s and 2000s. It is seen that the awards won by female writers in the field of Literature have doubled in the last 15 years. This indicator may be that the number of women writers is increasing in many countries.

### Men again have a high advantage in awards won in the medical field. However, there has been an increase in female scientists in the last 30 years. The rise in numbers of women in higher and general education worldwide may be one of the main reasons for this.

### Among the female winners in the field of peace, there have been a few younger winners in recent years. The events of the last 30 years, by young individuals who made actions that will leave a mark on human history, were rewarded and went down in history as women Nobel Peace Prize winners.

### It has won many times throughout the 115-year history of the men's Nobel Prize in Physics. Female scientists have won only twice. The presence of female thinkers in many areas of the world, even if they are few in number, is the first steps of development. The low success rates of women in this field may be due to the fact that there are not as many female students as thought in physics departments.

### If a general evaluation is made; It seems that the earnings of female scientists and intellectuals have increased in the last 30 years. We can predict that there will be an increase in the next 30-50 years. Wars, economic difficulties, migration and education are among the factors that directly affect the number of countries winning these awards.


# The categories and countries of the Nobel Prize winners between 1938 and 1945.

In [ ]:
nobel.dtypes

year              int64
category         object
prize            object
full_name        object
birth_date       object
birth_country    object
sex              object
dtype: object

In [ ]:
#The column was deleted by creating a new table from the table we created in the beginning. axis = 1 must be written because the command to delete it from top to bottom must be written.
#axis = 1 means ---> columns

nobel2 = nobel.drop(["prize","birth_date","sex","full_name"], axis = 1)

nobel2

year    category     birth_country
0    1901   Chemistry       Netherlands
1    1901  Literature            France
2    1901    Medicine  Prussia (Poland)
3    1901       Peace       Switzerland
4    1901       Peace            France
..    ...         ...               ...
906  2016    Medicine             Japan
907  2016       Peace          Colombia
908  2016     Physics    United Kingdom
909  2016     Physics    United Kingdom
910  2016     Physics    United Kingdom

[883 rows x 3 columns]

In [ ]:
nobel2.dtypes

year              int64
category         object
birth_country    object
dtype: object

In [ ]:
#filtering process was performed and data between the specified times was retrieved. The index was also rewritten.

nobel2_filtered_38_45 = nobel2[(nobel2['year'] >= 1938) & (nobel2['year'] <= 1945)].reset_index(drop=True)

nobel2_filtered_38_45

year    category              birth_country
0   1938   Chemistry  Austria-Hungary (Austria)
1   1938  Literature   United States of America
2   1938    Medicine                    Belgium
3   1938     Physics                      Italy
4   1939   Chemistry                    Germany
5   1939   Chemistry  Austria-Hungary (Croatia)
6   1939  Literature   Russian Empire (Finland)
7   1939    Medicine           Germany (Poland)
8   1939     Physics   United States of America
9   1943   Chemistry  Austria-Hungary (Hungary)
10  1943    Medicine                    Denmark
11  1943    Medicine   United States of America
12  1943     Physics           Germany (Poland)
13  1944   Chemistry                    Germany
14  1944  Literature                    Denmark
15  1944    Medicine   United States of America
16  1944    Medicine   United States of America
17  1944     Physics   Austria-Hungary (Poland)
18  1945   Chemistry   Russian Empire (Finland)
19  1945  Literature                      Chile
20  1945    Medicine                   Scotland
21  1945    Medicine                    Germany
22  1945    Medicine                  Australia
23  1945       Peace   United States of America
24  1945     Physics                    Austria

In [ ]:
nobel2_filtered_38_45.shape

(25, 3)

In [4]:
#bar chart used. The countries were brought to the x-axis and collected in categories and listed with different category colors on the countries.

fig3 = px.bar(nobel2_filtered_38_45, x="birth_country", color="category", title="1938-1945 WINNING CATEGORIES BY COUNTRY")
fig3.update_layout(
    title=dict(text="1938-1945 WINNING CATEGORIES BY COUNTRY", font=dict(size=24)),
)
fig3.show()

NameError: name 'px' is not defined

### Insight - European countries have a great overall advantage in the Nobel Prizes awarded between the start and end of World War II. The total number of awards received by the USA is equal to the total number of citizens of the Austro-Hungarian kingdom.
### It seems that the USA has single-handedly made a difference to other countries with the 3 different awards it has received in the Medicine category in these 7 years.
### The USA, which received awards in the categories of Peace, Physics, Literature and Medicine, is seen as the rising country of this period. It can be observed that the majority of scientists are those who migrated and escaped from the Second World War.


# The categories and countries of the Nobel Prize winners between 1947 and 1991.



In [ ]:
# filtering has been made.

nobel2_filtered_47_91 = nobel2[(nobel2['year'] >= 1947) & (nobel2['year'] <= 1991)].reset_index(drop=True)

nobel2_filtered_47_91

year    category                     birth_country
0    1947   Chemistry                    United Kingdom
1    1947  Literature                            France
2    1947    Medicine  Austria-Hungary (Czech Republic)
3    1947    Medicine  Austria-Hungary (Czech Republic)
4    1947    Medicine                         Argentina
..    ...         ...                               ...
372  1991  Literature                      South Africa
373  1991    Medicine                           Germany
374  1991    Medicine                           Germany
375  1991       Peace                   Burma (Myanmar)
376  1991     Physics                            France

[377 rows x 3 columns]

In [ ]:
#The highest earning country data in this date range was collected from highest to lowest and the table was printed.

nobel2_filtered_47_91_grouped = nobel2_filtered_47_91.groupby('birth_country').size().reset_index(name='Toplam').nlargest(10, 'Toplam')
nobel2_filtered_47_91_grouped

birth_country  Toplam
73  United States of America     116
72            United Kingdom      39
24                   Germany      30
21                    France      18
67                    Sweden      15
55                    Russia      10
35                     Italy       9
13                    Canada       8
68               Switzerland       8
36                     Japan       7

In [5]:
# It was thought that treemap would produce the best visual in order to group all 6 categories differently and compare them in graphics.
# All categories were evaluated differently and their totals were shown.

nobel2_filtered_47_91_grouped = nobel2_filtered_47_91.groupby(['category', 'birth_country']).size().reset_index(name='count')


fig5 = px.treemap(nobel2_filtered_47_91_grouped, path=['category', 'birth_country'], title="WINNING CATEGORIES BY COUNTRY 1947-1991",
                  values='count',
                  color='count',
                  hover_data=['birth_country'],
                  color_continuous_scale='Mrybm')

fig5.update_layout(margin=dict(t=65, l=45, r=25, b=45))
fig5.update_layout(title_x=0.5)
fig5.update_layout(
    title=dict(text="WINNING CATEGORIES BY COUNTRY 1947-1991", font=dict(size=24)),
)
fig5.show()


NameError: name 'nobel2_filtered_47_91' is not defined

### Insight - First of all, it was requested to examine the date range between 1947 and 1991. Since this interval was after World War II, this graph can be interpreted on a country basis, as the earning amounts of countries have an impact on the progress of development and the re-formation of society in many areas.
### At first glance, apart from the names of the categories, the prominent country names are the USA, Germany, the United Kingdom and France. It is among the top 5 countries in total numbers in the fields of Medicine, Physics, Chemistry and Economy.
### The fact that these countries are at the top in these matters may be due to the direct impact of the amount of immigration from around the world, especially to the USA, and the regular and open-minded education provided by the restructuring after the Second World War. His contributions to higher education and socio-cultural contributions have led to great developments in the development and progress of the whole in the fields of technology, science and economy.
### The top 10 in the fields of Literature and Peace has a different appearance than other branches of science. The awards received in literature include the UK, USA, France, Russia, Spain and Sweden. It can be seen that almost all of these awards received in the field of literature are from European countries and the USA is at the top due to the great immigration it received during these years.
### The fact that the Peace Prize has been won by many different countries can be said to have a great impact on Nobel Prizes and international political relations. Especially during these years when it was necessary to establish unity in Europe.







# The countries and ages of people after 2000 in the categories of Chemistry, Literature, Peace, Physics and Medicine.

In [ ]:
nobel1

year    category                                           prize  \
0    1901   Chemistry               The Nobel Prize in Chemistry 1901   
1    1901  Literature              The Nobel Prize in Literature 1901   
2    1901    Medicine  The Nobel Prize in Physiology or Medicine 1901   
3    1901       Peace                      The Nobel Peace Prize 1901   
4    1901       Peace                      The Nobel Peace Prize 1901   
..    ...         ...                                             ...   
906  2016    Medicine  The Nobel Prize in Physiology or Medicine 2016   
907  2016       Peace                      The Nobel Peace Prize 2016   
908  2016     Physics                 The Nobel Prize in Physics 2016   
909  2016     Physics                 The Nobel Prize in Physics 2016   
910  2016     Physics                 The Nobel Prize in Physics 2016   

                        full_name birth_date     birth_country   sex  \
0    Jacobus Henricus van 't Hoff 1852-08-30       Netherlands  Male   
1                 Sully Prudhomme 1839-03-16            France  Male   
2          Emil Adolf von Behring 1854-03-15  Prussia (Poland)  Male   
3               Jean Henry Dunant 1828-05-08       Switzerland  Male   
4                  Frédéric Passy 1822-05-20            France  Male   
..                            ...        ...               ...   ...   
906              Yoshinori Ohsumi 1945-02-09             Japan  Male   
907            Juan Manuel Santos 1951-08-10          Colombia  Male   
908             David J. Thouless 1934-09-21    United Kingdom  Male   
909          F. Duncan M. Haldane 1951-09-14    United Kingdom  Male   
910         J. Michael Kosterlitz 1943-06-22    United Kingdom  Male   

     winner_age  
0            49  
1            62  
2            47  
3            73  
4            79  
..          ...  
906          71  
907          65  
908          82  
909          65  
910          73  

[883 rows x 8 columns]

In [ ]:
#unnecessary columns were deleted again.

nobel3 = nobel1.drop(["prize","full_name","birth_date","sex"], axis = 1)

nobel3

year    category     birth_country  winner_age
0    1901   Chemistry       Netherlands          49
1    1901  Literature            France          62
2    1901    Medicine  Prussia (Poland)          47
3    1901       Peace       Switzerland          73
4    1901       Peace            France          79
..    ...         ...               ...         ...
906  2016    Medicine             Japan          71
907  2016       Peace          Colombia          65
908  2016     Physics    United Kingdom          82
909  2016     Physics    United Kingdom          65
910  2016     Physics    United Kingdom          73

[883 rows x 4 columns]

In [ ]:
#Filtering was done on 5 categories.

nobel3_filtered = nobel3[(nobel3['year'] >= 2000) & ((nobel3['category'] == "Chemistry") |
                                                      (nobel3['category'] == "Literature") |
                                                        (nobel3['category'] == "Peace") |
                                                         (nobel3['category'] == "Medicine") |
                                                          (nobel3['category'] == "Physics"))].reset_index(drop=True)

nobel3_filtered

year    category             birth_country  winner_age
0    2000   Chemistry  United States of America          64
1    2000   Chemistry               New Zealand          73
2    2000   Chemistry                     Japan          64
3    2000  Literature                     China          60
4    2000    Medicine                    Sweden          77
..    ...         ...                       ...         ...
157  2016    Medicine                     Japan          71
158  2016       Peace                  Colombia          65
159  2016     Physics            United Kingdom          82
160  2016     Physics            United Kingdom          65
161  2016     Physics            United Kingdom          73

[162 rows x 4 columns]

In [ ]:
#statistical description was printed to gain insight on ages.

nobel3_filtered.describe()

year  winner_age
count   162.000000  162.000000
mean   2007.802469   65.722222
std       5.022730   12.282012
min    2000.000000   17.000000
25%    2003.000000   58.000000
50%    2008.000000   67.000000
75%    2012.000000   75.000000
max    2016.000000   88.000000

In [ ]:
#Extra grouping was done, first by category, then by country of birth and age. But this data was not visualized.

nobel3_filtered_grouped = nobel3_filtered.groupby(['category', 'birth_country','winner_age']).size().reset_index(name='count')

nobel3_filtered_grouped

category                               birth_country  winner_age  count
0    Chemistry                                     Austria          83      1
1    Chemistry                                     Belgium          75      1
2    Chemistry       British Mandate of Palestine (Israel)          70      2
3    Chemistry       British Mandate of Palestine (Israel)          73      1
4    Chemistry  British Protectorate of Palestine (Israel)          57      1
..         ...                                         ...         ...    ...
148    Physics                    United States of America          77      1
149    Physics                    United States of America          79      1
150    Physics                    United States of America          80      1
151    Physics                    United States of America          88      1
152    Physics                      West Germany (Germany)          44      1

[153 rows x 4 columns]

In [ ]:
nobel3_filtered_grouped = nobel3_filtered.groupby(['category', 'birth_country']).agg({'winner_age': 'mean'}).reset_index()

fig11 = px.bar(nobel3_filtered_grouped, x='birth_country', y='winner_age', color='category', height=1200, width=3000,
             facet_row='category',
             category_orders={"category": ["Chemistry", "Literature", "Peace", "Physics", "Medicine"]},
             labels={'winner_age': 'Yaş', 'birth_country': 'Doğum Ülkesi'},
             title='Countries and Ages of Nobel Prize Winners After 2000')
fig11.update_layout(
    title=dict(text='Countries and Ages of Nobel Prize Winners After 2000', font=dict(size=24)),
)
fig11.show()


### Insight - the first thing that stands out is that the winners in almost all categories are over 50 years old.

### In this table, where the average age is 65, only those in their 20s are in the Peace Prize category. It seems that the winners of the award are thinkers and pioneers who were raised and educated in the great change that took place after the Second World War, from birth to higher education.

### There is a situation that can be seen as an interesting pattern in all award-winning countries: there are countries (European countries, North America, Far East) that have high world gross national product in the fields of science, chemistry, physics and medicine, and are strong for investment. Literature and Peace prizes are found in countries that have little or no power (Middle East, Africa and South America). We can think that the reason for this situation is political reasons.

### There are only two countries that have received awards in all branches. These appear to be China and the USA. As stated when making evaluations of countries, it may be the beginning of changes and restructurings in trade and production channels.  




# ---------  GENERAL EVALUATION  ------------

## It can be said that it is impossible not to think that the fact that Nobel Prizes go to different countries every year is directly related to the economic, social, environmental, political, technological and high or low welfare levels.

## The effects of the great changes experienced in European countries, the increase in trade channels at the economic level, the establishment of the United Nations and the increase in unity awareness, the rapid increase in production with information and new energy inventions, the easier accessibility of information in certain countries than in others, investment opportunities and development awareness, show the differences between countries. It can also be monitored in the results analysis of the awards.